commands/user.go

```diff
@@ -128,6 +137,7 @@ func init() {
 		ResetUserMfaCmd,
 		SearchUserCmd,
 		ListUsersCmd,
+		VerifyUserCmd,
```

Comment:
I'll change to `VerifyUserEmailCmd`

Reply:
I tried to stay consistent with the client method and server method therefore its `VerifyUser`. If we decide to change it, we can come up with a new name 👍

Reply:
Agree, we should follow the convention here. For `user verify-email` it would be `VerifyEmailUserCmd`, which is more descriptive